src/lib/lex.mll

```diff
@@ -39,7 +39,9 @@ let keywords =
     ("cof", COF);
     ("sub", SUB);
     ("path", PATH);
-    ("coe", COE)
+    ("coe", COE);
+    ("#t", TOPC);
```

Comment:
This `#t` and `#f` should be unreachable and deleted, right?

Reply:
whoops! yeah i didn't think about that; that makes sense.